nixos/modules/config/power-management.nix

```diff
@@ -2,6 +2,12 @@
 let
 
   cfg = config.powerManagement;
+  inherit (lib)
+    literalExpression
+    mkIf
+    mkOption
+    types
+    ;
```

Comment:
I personally do not think that this change improves anything really but if you really want to we can keep it.